src/database/queries/requestSessions.js

```diff
@@ -237,20 +237,23 @@ exports.getRequestSessions = async (requestSessionId) => {
 	}
 }
 
-exports.markRequestsAsDeleted = async (requestSessionIds = []) => {
+exports.markRequestsAsDeleted = async (requestSessionIds = [], tenantCode) => {
```

Comment:
@sumanvpacewisdom there are many functions in this file, we need to add tenantCode in all queries